src/lib/mongoBackend/compoundValueBson.h

```diff
@@ -37,14 +37,14 @@
 *
 * compoundValueBson (for objects) -
 */
-extern void compoundValueBson(std::vector<orion::CompoundValueNode*> children, mongo::BSONObjBuilder& b);
+extern void compoundValueBson(const std::vector<orion::CompoundValueNode*>& children, mongo::BSONObjBuilder& b);
```

Comment:
Different from the general rule with style fixing, maybe this kind of change (X -> const X&) should be mentioned in the CNR, e.g.:

"Fix: several changes in argument passing in mongoBackend library to align style, from X to const X&"